config/locales/transfer/tasks/conditions_met.en.yml

```diff
@@ -0,0 +1,27 @@
+en:
```

Comment:
I would leave off all but the absolute basic of content for this as I don't think it will be the same as conversions, like we did for the stakholder kick off task, coming back and adding the content later, when it's been done.